src/_gettsim/time_conversion.py

```diff
@@ -304,9 +306,11 @@ def _create_time_conversion_functions(
 
         missing_time_units = [unit for unit in all_time_units if unit != time_unit]
         for missing_time_unit in missing_time_units:
-            result[
-                f"{base_name}{missing_time_unit}{aggregation}"
-            ] = _create_function_for_time_unit(
+            new_name = f"{base_name}{missing_time_unit}{aggregation}"
```

Comment:
Could you add a comment on how this achieves the desired behavior? I can't wrap my head around it in 3min and this will only get worse once I have not thought about this problem the day before :laughing: